racket/src/worksp/msvcprep.bat

```diff
@@ -7,7 +7,12 @@ set VCMODE=%1
 set Applications=%ProgramFiles(x86)%
 if "%Applications%" == "" set Applications=%ProgramFiles%
 
-set VCVARBAT=%Applications%\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build\vcvarsall.bat
```

Comment:
Did this one get lost? I think it needs to be with the 2017 group as "if not exist ..."